helm/charts/hydra/templates/job-migration.yaml

```diff
@@ -57,7 +57,7 @@ spec:
         args: ["migrate", "sql", "-e", "--yes", "--config", "/etc/config/hydra.yaml"]
         {{- end }}
         env:
-        {{- if not (empty ( include "hydra.dsn" . )) }}
+        {{- if empty ( include "hydra.dsn" . ) }}
```

Comment:
The same behaviour is in the main deployment: https://github.com/ory/k8s/blob/master/helm/charts/hydra/templates/deployment.yaml#L122-L128